docs/MigrationTimeline.md

```diff
@@ -0,0 +1,101 @@
+# Migration Timelines
+
+## 15 day
+
+This 15-day migration timeline demonstrates an example of a migration process that can be adapted to 3 working weeks when including weekends. Key phases:
+
+1. Deployment and setup (Days 1-5)
+1. Historical backfill and catchup (Days 5-10)
+1. Validation and teardown (Days 11-15)
```

Comment:
Do we have confidence the average user will be able to have sufficient confidence in the new cluster that they'd be willing to cut over in just <5 days of testing?  I'm not sure we're providing them sufficient tools and techniques yet to make that possible considering the impact of getting that decision wrong.  I don't know what the "right" amount of time is for the average user, but 2 business weeks (or more) seems much more likely.

Reply:
I've had conversations with customers who'd like to have even less validation time given they have multiple stages. I'd like to keep it for now and include a preamble that discusses how there is no "one size fits most" timeline here